src/main/scala/tech/cryptonomic/conseil/Conseil.scala

```diff
@@ -31,115 +32,155 @@ object Conseil
     with ConseilOutput {
 
   loadApplicationConfiguration(args) match {
-    case Right((server, platforms, securityApi, verbose, metadataOverrides, nautilusCloud)) =>
+    case Left(errors) =>
```

Comment:
You can just have an empty case match with no error handling?

Reply:
I'm not sure I get the question...
Are you asking if you're not needed to actually put _some code_ in one of the `case` of the match?
The answer to this is "yes, as long as the return type of all branches of the match are consistent". Here consistent mean that they can be inferred to have a common shared super-type, that will be the return type of the whole match expression.

If the question was another, then I kindly ask you to clarify.